go/cmd/dolt/commands/sql.go

```diff
@@ -269,11 +269,9 @@ func sqlHandleVErrAndExitCode(queryist cli.Queryist, verr errhand.VerboseError,
 			if _, ok := queryist.(*engine.SqlEngine); !ok {
 				// We are in a context where we are attempting to connect to a remote database. These errors
 				// are unstructured, so we add some additional context around them.
-				tmpMsg := `You've encountered a new behavior in dolt which is not fully documented yet.
-A local dolt server is using your dolt data directory, and in an attempt to service your request, we are attempting to 
-connect to it. That has failed. You should stop the server, or reach out to @macneale on https://discord.gg/gqr7K4VNKe
-for help.`
-				cli.PrintErrln(tmpMsg)
+				remoteUnsupportedMsg := `This command has not yet been migrated to function in a remote context. Please shut down your 
+server and try again. Or, reach out to us on discord (https://discord.gg/gqr7K4VNKe) if you need help.`
+				cli.PrintErrln(remoteUnsupportedMsg)
```

Comment:
Let's simplify. Drop the first message entirely and change line 75 to:

`msg = fmt.Sprintf("Error connecting to remote database (%s): \"%s\".\n", host, msg)`

You can hard code the host to "localhost:<port>", but I'll need to update before too long with my changes in flight. Getting the port will be something you can do now. Ideas on how to make this easier?